### PR TITLE
Solve 05_sumAll

### PR DESCRIPTION
**Pull request by EdvinasStankunas**

### CODE REVIEW

File: 05_sumAll/sumAll.js [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+    typeof secArg != "number" ||
+    firstArg <= 0 ||
+    secArg <= 0
+  ) {
+    return "ERROR";
+  }
+  if (firstArg > secArg) {
+    const temp = firstArg;
+    firstArg = secArg;
+    secArg = temp;
```
Comment (arnassavickas) [inline 05_sumAll/sumAll.js]: You are modifying original arguments

File: 05_sumAll/sumAll.js [hunk 2]
```diff
@@ -1,6 +1,22 @@
-const sumAll = function() {
-
+const sumAll = function (firstArg, secArg) {
+  let sum = 0;
+  if (
+    typeof firstArg != "number" ||
```
Comment (arnassavickas) [inline 05_sumAll/sumAll.js]: Better become used to always using strict equality checks

File: 05_sumAll/sumAll.js [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+    typeof secArg != "number" ||
+    firstArg <= 0 ||
+    secArg <= 0
+  ) {
+    return "ERROR";
+  }
+  if (firstArg > secArg) {
```
Comment (arnassavickas) [inline 05_sumAll/sumAll.js]: <div></div>

File: 05_sumAll/sumAll.js [hunk 4]
```diff
@@ -1,6 +1,22 @@
-const sumAll = function() {
-
+const sumAll = function (firstArg, secArg) {
```
Comment (arnassavickas) [inline 05_sumAll/sumAll.js]: Of course they are arguments, try to find a name that would better describe type or usage of the value.

File: 05_sumAll/sumAll.js (hunk 1 above)
Comment (arnassavickas) [reply 05_sumAll/sumAll.js]: You could read about `Math.min`, and `Math.max`, should also help.

File: 05_sumAll/sumAll.js (hunk 4 above)
Comment (EdvinasStankunas) [reply 05_sumAll/sumAll.js]: Would "rangeStart" and "rangeEnd" be appropriate?

File: 05_sumAll/sumAll.js [hunk 5]
```diff
@@ -1,6 +1,28 @@
-const sumAll = function() {
+const sumAll = function (rangeStart, rangeEnd) {
+  let sum = 0;
+  if (
```
Comment (arnassavickas) [inline 05_sumAll/sumAll.js]: Just this one little space
```suggestion
  let sum = 0;
  
  if (
```

**Review by arnassavickas — APPROVED** [review]
LGTM 👌

**Pull request merged**